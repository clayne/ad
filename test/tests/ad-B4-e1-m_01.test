ad | -c always -B4 -e1 -m | endian.bin | | 0
