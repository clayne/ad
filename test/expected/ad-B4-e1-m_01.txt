[32m[K0000000000000240[m[K[36m[K:[m[K [41;1m[K0100[m[K [41;1m[K0000[m[K [41;1m[K[m[KFFFF FFFF  FFFF FFFF FF[41;1m[K01[m[K [41;1m[K0000[m[K  [41;1m[K....[m[K.........[41;1m[K...[m[K
[32m[K0000000000000250[m[K[36m[K:[m[K [41;1m[K00[m[KFF FFFF FFFF FFFF  FFFF FFFF FFFF [41;1m[K0100[m[K  [41;1m[K.[m[K.............[41;1m[K..[m[K
[32m[K0000000000000260[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K[m[KFFFF FFFF FFFF  FFFF FFFF FFFF FF[41;1m[K01[m[K  [41;1m[K..[m[K.............[41;1m[K.[m[K
[32m[K0000000000000270[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K00[m[KFF FFFF FFFF  FFFF FFFF FFFF FFFF  [41;1m[K...[m[K.............
[32m[K0000000000000300[m[K[36m[K:[m[K [41;1m[K0100[m[K [41;1m[K0000[m[K [41;1m[K[m[K0000 0000  FF[41;1m[K01[m[K [41;1m[K0000[m[K [41;1m[K00[m[K00 0000  [41;1m[K....[m[K.....[41;1m[K....[m[K...
[32m[K0000000000000310[m[K[36m[K:[m[K 00FF FFFF FFFF FFFF  FFFF [41;1m[K0100[m[K [41;1m[K0000[m[K [41;1m[K[m[K0000  ..........[41;1m[K....[m[K..
[32m[K0000000000000320[m[K[36m[K:[m[K 0000 FFFF FFFF FFFF  FFFF FF[41;1m[K01[m[K [41;1m[K0000[m[K [41;1m[K00[m[K00  ...........[41;1m[K....[m[K.
[32m[K0000000000000330[m[K[36m[K:[m[K 0000 00FF FFFF FFFF  FFFF FFFF [41;1m[K0100[m[K [41;1m[K0000[m[K  ............[41;1m[K....[m[K
[32m[K0000000000000340[m[K[36m[K:[m[K 0000 0000 FFFF FFFF  FFFF FFFF FF[41;1m[K01[m[K [41;1m[K0000[m[K  .............[41;1m[K...[m[K
[32m[K0000000000000350[m[K[36m[K:[m[K [41;1m[K00[m[K00 0000 00FF FFFF  FFFF FFFF FFFF [41;1m[K0100[m[K  [41;1m[K.[m[K.............[41;1m[K..[m[K
[32m[K0000000000000360[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K[m[K0000 0000 FFFF  FFFF FFFF FFFF FF[41;1m[K01[m[K  [41;1m[K..[m[K.............[41;1m[K.[m[K
[32m[K0000000000000370[m[K[36m[K:[m[K [41;1m[K0000[m[K [41;1m[K00[m[K00 0000 00FF  FFFF FFFF FFFF FFFF  [41;1m[K...[m[K.............
